### PR TITLE
allow spaces in options.binary

### PR DESCRIPTION
**Pull request by onelesd**
This allows using network namespaces w/ phantomjs:

```
phantomOpts = {
  hostname: '4.3.2.10',
  binary: "ip netns exec ns4-3-2-10 /usr/local/bin/phantomjs"
}
phantom.create(phantomOpts, function (ph){...})
```

### CODE REVIEW

File: phantom.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-// Generated by CoffeeScript 1.9.2
```
Comment (amir20) [inline phantom.js]: Why did you down grade coffee script? Can you use at least the same version please?

**Comment by amir20** [comment]
Update coffee script version and I can merge

File: phantom.js (hunk 1 above)
Comment (onelesd) [inline phantom.js]: Sorry about that. I didn't check it and Debian installed 1.4.0.

**Comment by amir20** [comment]
looks good.

**Pull request merged**